src/chainparamsbase.cpp

```diff
@@ -103,8 +103,3 @@ std::string ChainNameFromCommandLine()
         return CBaseChainParams::TESTNET;
     return CBaseChainParams::MAIN;
 }
-
-bool AreBaseParamsConfigured()
```

Comment:
ACK on this one, apparently the last use of this went away in #6149/27d760580456d206c5a02ef29797f296f510099c.